modules/notifications/assets/js/components/whats-new.js

```diff
@@ -0,0 +1,264 @@
+					<IconButton
+						aria-label={ 'close' }
+						size="small"
+						onClick={ () => setIsOpen( false ) }
+					>
+						<XIcon />
+					</IconButton>
+				</Toolbar>
+			</AppBar>
+			<Divider />
+		</>
+	);
+};
+
+const WhatsNewDrawerContent = () => {
+	const { isPending, error, data: items } = useQuery({
+		queryKey: [ 'e-notifications' ],
+		queryFn: getNotifications,
+	} );
+
+	if ( isPending ) {
+		return (
+			<Box>Loading...</Box>
+		);
+	}
+
+	if ( error ) {
+		return (
+			<Box>
+				An error has occurred: { error }
+			</Box>
+		);
+	}
+
+	return (
+		items.map( ( item, itemIndex ) => {
+			const chips = [];
+
+			if ( item.chipPlan ) {
+				chips.push( {
+					color: 'accent',
+					label: item.chipPlan,
+				} );
+			}
+
+			if ( item.chipTags.length ) {
+				item.chipTags.forEach( ( chipTag ) => {
+					chips.push( {
+						variant: 'outlined',
+						label: chipTag,
+					} );
+				} );
+			}
+
+			return (
+				<Box
+					key={ itemIndex }
+					display="flex"
+					flexDirection="column"
+					sx={ {
+						paddingBlockStart: 2,
+					} }
+				>
+					<Stack
+						direction="row"
+						divider={ <Divider orientation="vertical"
+						                   flexItem/> }
+						spacing={ 1 }
+						color="text.tertiary"
+						sx={ {
+							paddingBlockEnd: 1,
+						} }
+					>
+						{ item.topic && (
+							<Box>{ item.topic }</Box>
+						) }
+						<Box>{ item.date }</Box>
+					</Stack>
+					<WrapperWithLink link={ item.link }>
+						<Typography
+							variant="subtitle1"
+							sx={ {
+								paddingBlockEnd: 2,
+							} }
+						>
+							{ item.title }
+						</Typography>
+					</WrapperWithLink>
+					{ item.imageSrc && (
+						<Box
+							sx={ {
+								paddingBlockEnd: 2,
+							} }
+						>
+							<WrapperWithLink link={ item.link }>
+								<img
+									src={ item.imageSrc }
+									alt={ item.title }
+									style={ { maxWidth: '100%' } }
+								/>
+							</WrapperWithLink>
+						</Box>
+					) }
+					{ chips && (
+						<Stack
+							direction="row"
+							flexWrap="wrap"
+							gap={ 1 }
+							sx={ {
+								paddingBlockEnd: 1,
+							} }
+						>
+							{ chips.map( ( chip, chipIndex ) => {
+								return (
+									<Chip
+										key={ `chip-    ${ itemIndex }${ chipIndex }` }
+										{ ...chip }
+									/>
+								);
+							} ) }
+						</Stack>
+					) }
+					<Typography
+						variant="body2"
+						sx={ {
+							paddingBlockEnd: 2,
+						} }
+					>
+						{ item.description }
+						{ item.readMoreText && (
+							<>
+								{ ' ' }
+								<Link
+									href={ item.link }
+									target="_blank"
+								>
+									{ item.readMoreText }
+								</Link>
+							</>
+						) }
+					</Typography>
+					{ item.cta && item.ctaLink && (
+						<Box
+							sx={ {
+								paddingBlockEnd: 2,
+							} }
+						>
+							<Button
+								href={ item.ctaLink }
+								target="_blank"
+								variant="contained"
+								color="accent"
```

Comment:
This color is going to be deprecated, maybe let's upgrade the `@elementor/ui` version?

Reply:
The UI version has been updated, you can pull from "main" and use `promotion` instead of `accent`.